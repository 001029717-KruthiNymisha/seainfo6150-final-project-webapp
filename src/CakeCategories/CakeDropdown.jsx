import React from "react";
import { Link } from "react-router-dom";
import styles from "./Categories.module.css";

const CakeDropdown = function () {
  return (
    <div className={styles.navbar}>
      <div className={styles.dropdown}>
        <button className={styles.dropbtn}>
          Categories
        </button>
        <div className={styles.dropdownContent}>
          <Link to="/Cakes" className={styles.subCategoryItem}>
                      Cakes
                    </Link>
          <Link to="/WeddingCakes" className={styles.subCategoryItem}>
            Wedding Cakes
          </Link>
          <Link to="/GlutenFreeCakes" className={styles.subCategoryItem}>
            Gluten-Free Cakes
          </Link>
          <Link to="/BirthdayCakes" className={styles.subCategoryItem}>
           BirthdayCakes
          </Link>
        </div>
        </div>
        </div>
  );
};

export default CakeDropdown;
