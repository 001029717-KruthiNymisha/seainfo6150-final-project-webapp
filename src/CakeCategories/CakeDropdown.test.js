import React from "react";
import { render } from "@testing-library/react";
import CakeDropdown from "./CakeDropdown.jsx";
import { BrowserRouter } from "react-router-dom";

it("renders the CakeCategories component correctly", () => {
  const { container } = render(
    <BrowserRouter>
      <CakeDropdown />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});