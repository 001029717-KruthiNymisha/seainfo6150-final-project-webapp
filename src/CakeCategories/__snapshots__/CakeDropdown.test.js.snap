// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CakeCategories component correctly 1`] = `
<div>
  <div
    class="navbar"
  >
    <div
      class="dropdown"
    >
      <button
        class="dropbtn"
      >
        Categories
      </button>
      <div
        class="dropdownContent"
      >
        <a
          class="subCategoryItem"
          href="/Cakes"
        >
          Cakes
        </a>
        <a
          class="subCategoryItem"
          href="/WeddingCakes"
        >
          Wedding Cakes
        </a>
        <a
          class="subCategoryItem"
          href="/GlutenFreeCakes"
        >
          Gluten-Free Cakes
        </a>
        <a
          class="subCategoryItem"
          href="/BirthdayCakes"
        >
          BirthdayCakes
        </a>
      </div>
    </div>
  </div>
</div>
`;
